Manual/manJHUGenerator.tex

```diff
@@ -1154,6 +1157,33 @@ \subsection{LHAPDF}
 \end{itemize}
 %%%%%%%%%%%
 
+\subsection{Including weight for decay of a stable wide resonance}
+In decay of a BSM Higgs boson with a large width, e.g. generated by POWHEG, it might be important 
+to account for the final state phase space, in addition to generating the actual final state particle,
+such as $H\to ZZ\to 4f$ or $H\to WW\to 4f$. This could be configured with a weight discussed below. 
+In addition, correction to the propagator could be applied in this step, such as for example the 
+CPS scheme adopted in the POWHEG generator. To use these options, 
+add the following command line options in the JHUGen card: 
+\begin{verbatim}
+ReadLHE=(input file).lhe ReweightDecay WidthSchemeIn=3 ReadPmHstar PmHstarFile=(filename.out)
+\end{verbatim}
+The weight distributions depend strongly on the $HVV$ couplings. 
+Weights generated for Standard Model couplings would not be applicable to BSM couplings. 
+One can get the weight as a function of $m_{VV}$ by running ($WW$ example below):
+\begin{verbatim}
+./JHUGen DecayMode1=10 DecayMode2=10 MReso=(mass) PrintPMZZ=min,max PMZZIntervals=intervals 
```

Comment:
What about the interference setting? I thought Heshy was suggesting to turn it off, was it not the case?

Reply:
this should be left to be configured by the user, as below threshold interference is important.

Reply:
Sure, but my recollection was that @hroskes saw numerical instabilities at lower masses (insufficient integration), so he wasn't actually extending the range to lower masses. @hroskes do you remember these issues, what is your opinion?

Reply:
This has nothing to do with approving this request, lets move forward

Reply:
No it does. If one writes instructions on this use case, one should also mention if numerical instabilities are expected, no algorithm is free from numerical instabilities. That is why it is important that we remember what was done previously.

Reply:
Do you want to take another 24 hours to wait? If yes, then please approve it in 24 hours, and we will all wait again.

Reply:
The 24-hour rule is to submit a review, not to approve (and I submitted my reviews within 24 hrs of being requested to review). Whether approval happens within 24 h depends on the people making the PR.

Reply:
This is nonsense discussion which should not happen in Git. What I mean is what do you want to happen: I can wait, you can wait, but at some point we need to move forward. How much time do you need to move forward? I do not expect either of us will have more information to contribute.